tests/base.test.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  getToken: () => 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjU0NmRhOTFlLTQ0ZmYtNGUxMS1hY2JkLTg4ZTE0MmY4NDJjNCIsImVtYWlsIjoidmljZW50QGdtYWlsLmNvbSIsImV4cCI6MTU1NDg4OTUzMS4yMjEsImlhdCI6MTU1NDg4NTkzMX0.n5xnezd87eoLOCy_FYcVtrBftPCNXxpacXTIlGeA0TY'
```

Comment:
Better if you don't expose your token outside.

Reply:
It was for test purpose